src/Microsoft.ML.Transforms/OneHotEncoding.cs

```diff
@@ -141,8 +143,55 @@ internal OneHotEncodingTransformer(ValueToKeyMappingEstimator term, IEstimator<I
         IRowToRowMapper ITransformer.GetRowToRowMapper(DataViewSchema inputSchema) => ((ITransformer)_transformer).GetRowToRowMapper(inputSchema);
     }
     /// <summary>
-    /// Estimator which takes set of columns and produce for each column indicator array.
+    /// Converts one or more input columns of categorical values into as many ourput columns of one-hot encoded vectors.
```

Comment:
output

Reply:
Thank you! I will download the spell check for VS on my laptop and make sure I don't have more typos